templates/admin/group/group_property_editor.html.ep

```diff
@@ -23,6 +23,17 @@
                     </div>
                 </div>
                 <div class="form-group">
+                    <label for="editor-build-version-sort" class="col-sm-2 control-label" data-toggle="tooltip" title="Whether BUILD values for this group sort as version numbers">Sort BUILDs as versions?</label>
+                    <div class="col-sm-10">
+                        <input type="radio" id="editor-build-version-sort" name="build_version_sort" value="1"
+                            %= 'checked' if $group->build_version_sort
+                            > Yes
```

Comment:
The yes/no text should be the label for the radio button, not the text at the left. But the use of a radio button should be ok in general.